staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_common.go

```diff
@@ -289,3 +289,23 @@ func (c *controllerCommon) DisksAreAttached(diskNames []string, nodeName types.N
 
 	return attached, nil
 }
+
+func filterDetachingDisks(unfilteredDisks []compute.DataDisk) []compute.DataDisk {
+	filteredDisks := []compute.DataDisk{}
+	for _, disk := range unfilteredDisks {
+		if disk.ToBeDetached != nil {
+			if *disk.ToBeDetached {
+				if disk.Name != nil {
```

Comment:
since disk is filtered, log level should be V(2), need to call out

Reply:
Looks like I suggested V(12) in the above comment, maybe V(2) is more appropriate since attach/detach does not always happen, we may need to call out, sorry for the confusion.